<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QMainWindow" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>477</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_netctlAuto" native="true">
      <layout class="QHBoxLayout" name="layout_netctlAuto">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_netctlAuto">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>netctl-auto is running</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_netctlAuto">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_main">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>4</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="textAlignment">
        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
       </property>
      </column>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main actions toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNetctl_auto"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop_all"/>
   <addaction name="separator"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
  </widget>
  <action name="actionNetctl_auto">
   <property name="text">
    <string>netctl-auto</string>
   </property>
   <property name="toolTip">
    <string>Show netctl-auto window</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start or stop selected profile</string>
   </property>
  </action>
  <action name="actionSwitch">
   <property name="text">
    <string>Switch</string>
   </property>
   <property name="toolTip">
    <string>Switch to selected profile</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="toolTip">
    <string>Restart selected profile</string>
   </property>
  </action>
  <action name="actionEnable">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
   <property name="toolTip">
    <string>Enable or disable selected profile</string>
   </property>
  </action>
  <action name="actionStop_all">
   <property name="icon">
    <iconset theme="process-stop">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Stop all</string>
   </property>
   <property name="toolTip">
    <string>Stop all profiles</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset theme="document-edit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit selected profile</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove selected profile</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
